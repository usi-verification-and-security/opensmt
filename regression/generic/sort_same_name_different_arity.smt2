(set-logic QF_UF)
(declare-sort U 0)
(declare-sort U 1)
