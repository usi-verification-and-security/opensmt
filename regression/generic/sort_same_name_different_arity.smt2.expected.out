(error "sort U already declared")

